src/appmixer/square/core/FindCustomers/component.json

```
@@ -108,19 +102,25 @@
                         "index": 7,
                         "multiple": true,
                         "options": [
-                            { "label": "Third Party", "value": "THIRD_PARTY" },
-                            { "label": "Square", "value": "SQUARE" },
-                            { "label": "Square Invoices", "value": "INVOICES" },
-                            { "label": "Square Customers API", "value": "CUSTOMERS_API" },
-                            { "label": "Square Dashboard", "value": "DASHBOARD" },
-                            { "label": "Square Point of Sale", "value": "SQUARE_POS" },
-                            { "label": "Terminals API", "value": "TERMINALS_API" },
-                            { "label": "Terminal", "value": "TERMINAL" },
-                            { "label": "Billing", "value": "BILLING" },
-                            { "label": "Bookings", "value": "BOOKINGS" },
-                            { "label": "Marketing", "value": "MARKETING" },
+                            { "label": "Other", "value": "OTHER" },
+                            { "label": "Appointments", "value": "APPOINTMENTS" },
+                            { "label": "Coupon", "value": "COUPON" },
```

```suggestion
                            { "clearItem": true, "label": "<empty>"},
                            { "label": "Other", "value": "OTHER" },
                            { "label": "Appointments", "value": "APPOINTMENTS" },
                            { "label": "Coupon", "value": "COUPON" },
```
It should also have an option to unselect the value using the `"clearItem": true` as the selection is not required and I might want to not filter it by any Creation Source

---

That's a great point, Let me try and add for all non required select fields if possible.

Thanks!